<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionView</class>
 <widget class="QDialog" name="ActionView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ActionView</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Mao de 11: Aceitar ou Correr ?</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Aceitar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Correr</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
